project.clj

```diff
@@ -1,11 +1,10 @@
-(defproject tech.droit/malcontent "0.1.0"
+(defproject tech.droit/malcontent "0.2.0-SNAPSHOT"
   :description "Ring middleware for HTTP Content Security Policy"
   :url "http://github.com/droitfintech/malcontent"
   :license {:name "Apache License, Version 2.0"
             :url "https://www.apache.org/licenses/LICENSE-2.0.html"}
-  :dependencies [[org.clojure/clojure "1.5.0"]
-                 [ring "1.2.0"]
-                 [net.sf.uadetector/uadetector-resources "2013.02"]]
-  :profiles {:dev {:dependencies [[speclj "2.5.0"]]}}
-  :plugins [[speclj "2.5.0"]]
+  :dependencies [[org.clojure/clojure "1.10.3"]
+                 [net.sf.uadetector/uadetector-resources "2014.10"]]
```

Comment:
I looked into the changes for this version:
https://github.com/arouel/uadetector/compare/2013.02...2014.10

I don't see a changelog anywhere unfortunately for a simple summary.

It overall looks like the changes were just more additions to what it can detect and a lot of extra handling of robot-script headers etc I believe. Definitely a lot to look at, but nothing stands out as a breaking change to worry about. This was a well known and used lib at the time too, so it makes sense to keep up to date. I do think in the future something else likely will need to be used if this project has no active forks.
It's interesting to note that this dependency seems very dated (year 2014 is the latest). However, for now it seems to be ok.

I have also noticed that a few classes imported from the project [do not seem to be used here](https://github.com/droitfintech/malcontent/blob/8ca45e43b01c563d21d5b137078f366a1cc1f11c/src/malcontent/browser.clj#L2).

Reply:
> I have also noticed that a few classes imported from the project [do not seem to be used here](https://github.com/droitfintech/malcontent/blob/8ca45e43b01c563d21d5b137078f366a1cc1f11c/src/malcontent/browser.clj#L2).

Addressed lint warnings in d4497f7.